data/data/azure/vnet/common.tf

```diff
@@ -13,5 +27,6 @@ locals {
 
   public_lb_id   = azurerm_lb.public.id
   internal_lb_id = azurerm_lb.internal.id
-}
 
+  virtual_network = var.virtual_network
```

Comment:
I don't see where this is used other than output. Perhaps consolidate all into output?

Reply:
I now see where it is 'like gcp'. It was used there because the `subnet` creation was handled in the `vpc` module. However, here that is done in the main module. This would need to stay if it gets moved down into the `vnet` module like recommended elsewhere. Otherwise it can be removed and output can use `var.virtual_network` instead.